docs/contributing/deploying-the-review-app.md

```diff
@@ -0,0 +1,48 @@
+# Deploying the review app
+
+GOV.UK Frontend review app is automatically deployed on Heroku for PRs raised from branches of the `alphagov/govuk-frontend` repository.
+If you've forked GOV.UK Frontend in your own repository (for contributing a bug fix or a feature, for example),
+you may want to deploy the review app as well, for example to share components preview with other people while working.
+This will not happen automatically and you'll need to set up your own deployment for your repository.
+
+## Hosting services
+
+GOV.UK Frontend review app runs on hosting services that supports Node.js.
+This means it does not run on 'static' hosting services like GitHub Pages or Netlify.
+
+Your organisation may already use such hosting service (for the Prototype Kit, for example).
+Check with your IT or digital team about which platform to use.
+
+Some hosting services automatically publish every time you push to GitHub. For example:
+
+- [Railway](https://railway.app/new/github)
+- [Render](https://render.com/docs/github)
+- [Heroku](https://devcenter.heroku.com/articles/github-integration) (requires payment)
```

Comment:
**note** Taken from [the Prototype Kit docs](https://prototype-kit.service.gov.uk/docs/publishing#hosting-services). I don't think we need to put much effort in maintaining that list, just give pointers for people to start doing their own research.